data_model/master/clusters/MeterIdentification.xml

```diff
@@ -1,6 +1,7 @@
-<?xml version="1.0"?>
+<?xml version="1.0"?>
 <!--
-Copyright (C) Connectivity Standards Alliance (2021). All rights reserved.
+Copyright (C) Connectivity Standards Alliance (2024). All rights reserved.

```

Comment:
I expect these to be updated via spec import only. How was this updated?

Reply:
And also why? Why is every line change? I would have expected them to be the same. Was this a windows vs linux newline change? please preserve newline style.

Reply:
Fixed